components/button/button-icon.js

```diff
@@ -117,7 +118,8 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 					box-shadow: var(--d2l-button-icon-focus-box-shadow);
 				}
 
-				.d2l-button-icon {
+				d2l-icon,
+				slot[name="icon"]::slotted(d2l-icon-custom) {
```

Comment:
This works eh? Like you don't need `slot[name="icon"]::slotted(d2l-icon)` after it gets re-slotted?

Reply:
It so far seems fine but I can update it just in case

Reply:
To test this out, you could do something like this:

```css
d2l-icon { width: 2em; }
slot[name="icon"]::slotted(d2l-icon) { width: 5em; }
```

And then see which one is getting applied in dev-tools, or see if maybe both are?

Reply:
Ah yup only `d2l-icon` gets used. Checked on Chrome, Safari, and Firefox and all use the `d2l-icon` css, the other doesn't seem to get applied at all.

Reply:
Ok cool so your original code was correct -- sorry for sending you down that path!

Reply:
No worries!